src/Core/Mews.Fiscalizations.Core/Xml/XmlSerializationData.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FuncSharp;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Mews.Fiscalizations.Core.Xml
+{
+    public sealed class XmlSerializationData
+    {
+        public XmlSerializationData(IEnumerable<XmlNamespace> namespaces = null)
+            : this(Encoding.UTF8, namespaces)
+        {
+        }
+
+        public XmlSerializationData(Encoding encoding, IEnumerable<XmlNamespace> namespaces = null)
```

Comment:
So instead of having 2 constructors, let's make the encoding optional and defaulting to UTF 8